angular/base/package.json

```diff
@@ -20,6 +20,7 @@
     "@angular/platform-browser": "6.0.0",
     "@angular/platform-browser-dynamic": "6.0.0",
     "@angular/router": "6.0.0",
+    "@angular/service-worker": "^6.0.0",
```

Comment:
Let's be precise with Angular dependencies. `^6.0.0` grabs a scary amount of version ranges

Reply:
keywords being "should not" 

It's not a good experience to ship starters with these version ranges and having it break if a mistake was made on Angular's end. We're on cutting edge technology, so it's safer to lock in versions and test the starters before deploying them to everyone.

Reply:
Plus, we use dependabot now, so upgrades to latest versions happen quite quickly, as long as CI is passing